charts/plugin-site-issues/templates/secret.yaml

```diff
@@ -10,5 +10,5 @@ data:
   jira_url: {{ required "jira.url is required" .Values.jira.url | toString | b64enc }}
   jira_username: {{ required "jira.username is required" .Values.jira.username | toString | b64enc }}
   jira_password: {{ required "jira.password" .Values.jira.password | toString | b64enc }}
-  github_app_id: {{ required "github.app_id is required" .Values.github.app_id | toString | b64enc }}
-  github_app_private_key: {{ required "github.app_id is required" .Values.github.app_private_key | toString | b64enc }}
+  github_app_id: {{ required "github.appId is required" .Values.github.appId | toString | b64enc }}
+  github_app_private_key: {{ required "github.appPrivateKey is required" .Values.github.appPrivateKey | toString | b64enc }}
```

Comment:
@halkeye the field used in the required message need to be replaced

Reply:
Yea I stared to fix secrets when I saw that. 

What's the convention? If no convention let's just update this chart and everything in else should work

Reply:
No convention afaik, and since the chart have to be updated I would fix all there instead of in charts-secrets. (And keep the same values fields names alongs related charts)